tfx/components/example_gen/import_example_gen/executor_test.py

```diff
@@ -140,6 +158,32 @@ def testDoWithProto(self):
         self.examples.get_string_custom_property(
             utils.PAYLOAD_FORMAT_PROPERTY_NAME))
 
+  def testDoWithSequenceExamples(self):
+    self._input_config = json_format.MessageToJson(
+        example_gen_pb2.Input(splits=[
+            example_gen_pb2.Input.Split(name='tfrecord_sequence', 
+                                        pattern='tfrecord_sequence/*'),
+        ]),
+        preserving_proto_field_name=True)
+        
+    exec_properties = {
+        utils.INPUT_BASE_KEY:
+            self._input_data_dir,
+        utils.INPUT_CONFIG_KEY:
+            self._input_config,
+        utils.OUTPUT_CONFIG_KEY:
+            self._output_config,
+        utils.OUTPUT_DATA_FORMAT_KEY:
+            example_gen_pb2.PayloadFormat.FORMAT_TF_SEQUENCE_EXAMPLE,
```

Comment:
just confirm, if change to format tf example, will it fail?

Reply:
no it doesn't fail.

but the same thing applies to testImportExample -- if I change format to tf sequence example, it doesn't fail.